P/Python/build_tarballs.jl

```diff
@@ -102,14 +103,17 @@ products = Product[
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("Expat_jll"; compat="2.2.10"),
-    Dependency("Bzip2_jll"; compat="1.0.8"),
-    Dependency("Libffi_jll"; compat="~3.2.2"),
+    Dependency("Expat_jll"; compat="2.6.5"),
+    Dependency("Bzip2_jll"; compat="1.0.9"),
+    Dependency("Libffi_jll"; compat="3.4.6"),
```

Comment:
I believe libffi breaks ABI in every minor version, reason why there was a `~` there that you removed.